src/sdgio-edit.owl

```diff
@@ -1596,6 +1601,15 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020237> "sub-outcome of an intermediate development outcome"@en)
 SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020237> <http://purl.unep.org/sdg/SDGIO_00000251>)
 
+# Class: <http://purl.unep.org/sdg/SDGIO_00020300> (cross-cutting objective)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020300> "A CGIAR objective specification which directs the organisation's mission such that 1) it contributes to the United Nation’s Sustainable Development Goals and 2) each research group contributes to its achievement.")
```

Comment:
This definition doesn't differentiate this kind of objective from any others. What does it mean to be "cross-cutting" in the CGIAR context?

Reply:
I intended the part "2) each research group contributes to its achievement" to differentiate the CCO class from the SLO. 

The part "[it] directs the organisation's mission such that it contributes to the UN's SDGs" intended differentiate the CCOs from intermediate and so-called sub-intermediate objectives.

Does the definition still seem poor?

Reply:
>Does the definition still seem poor?

Yes, but I think this is because we need more guidance from the CG strategic team. 

It's still not clear what makes this cross-cutting. What research groups are being referred to?

Reply:
Fix typo on "United Nation's" here and check for other such errors.

Reply:
Fixed here and in the definition of the SLO class.

Reply:
I don't see the typo fix. I'll do it in the next commit.


There's still no clarity on what makes the issue cross-cutting. Please create a separate issue.

Reply:
I created issues #168.